builds/fabric8-che/plugins/keycloak-plugin-token-provider/src/main/java/com/redhat/che/keycloak/token/provider/contoller/TokenController.java

```diff
@@ -117,6 +121,7 @@ public Response getOpenShiftToken(@HeaderParam(HttpHeaders.AUTHORIZATION) String
 
     @GET
     @Path("/user")
+    @ApiOperation(value = "Check if passed keycloak token matches token assigned to OSIO user who owns current namespace.")
```

Comment:
could the description be something like that ?
https://github.com/redhat-developer/rh-che/pull/173#issue-240872944

> which returns true if the token provided in the authorization header corresponds to the user who owns the namespace.

?